src/lib/parsers/include/sol-json.h

```diff
@@ -480,6 +480,181 @@ int sol_json_token_get_unescaped_string(const struct sol_json_token *token, stru
 char *sol_json_token_get_unescaped_string_copy(const struct sol_json_token *value);
 
 /**
+ * @brief Get the value of the JSON Object child element pointed by @a key_slice.
+ *
+ * @param scanner An initialized scanner, which is pointing to a JSON Object.
+ * @param key_slice The key of the desired element.
+ * @param value A pointer to the structure that will be filled with the value
+ *        of the element pointed by @a key_slice.
+ *
+ * @return If any parameter is invalid, or if @a scanner is not pointing to a
+ *         JSON Object, return -EINVAL. If @a key_slice is not present in the
+ *         JSON Object, -ENOENT. If @a key was found, and @a value was update
+ *         successfully with the value pointed by @a key, return 0.
+ *
+ * @see sol_json_array_get_at_index()
+ */
+int sol_json_object_get_key(struct sol_json_scanner *scanner, const struct sol_str_slice key_slice, struct sol_json_token *value);
```

Comment:
The doc says this gets the value, but the name says it gets the key, isn't this confusing?